keyboards/melgeek/z70ultra/z70ultra.c

```diff
@@ -116,7 +117,104 @@ led_config_t g_led_config = {
 		                                                               4,        4,          4,         4,         4,          4,        4,         4,
 	}
 };
+#endif
+#ifdef REV2
```

Comment:
Everything from Lines 121-217 should be placed in a keyboards/melgeek/z70ultra/rev2/rev2.c file.